Magento2/Tests/GraphQL/ValidFieldNameUnitTest.graphqls

```diff
@@ -0,0 +1,31 @@
+type FooType {
+    # Valid snake case field names
+    valid_snake_case_field: String
```

Comment:
We may have to change this rule soon to enforce camel case as discussed on architecture meeting. Please comment out this rule for now (including the check for interfaces).

Reply:
Rule was commented out, see https://github.com/magento/magento-coding-standard/pull/141/commits/2a8d52183bb612c3e4b520fb9b6d8a664932d9a9